src/components/loginForm/LoginForm.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+import "./LoginForm.css";
+import LoginButton from "../LoginButton";
+
+function LoginForm() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    try {
+      const response = await fetch("/api/login", {
```

Comment:
We will keep a generic method for API calls and will avoid using fetch directly inside every component